configs/twins/twins_pcpvt-s_fpn_fpnhead_8x4_512x512_80k_ade20k.py

```diff
@@ -0,0 +1,9 @@
+_base_ = [
+    '../_base_/models/twins_pcpvt-s_fpn.py', '../_base_/datasets/ade20k.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_80k.py'
+]
+# model settings
+norm_cfg = dict(type='SyncBN', requires_grad=True)
+
+optimizer = dict(_delete_=True, type='AdamW', lr=0.0001, weight_decay=0.0001)
```

Comment:
The setting is different from configs/twins/twins_pcpvt-l_uperhead_8x2_512x512_160k_ade20k.py.
Please have a check.

Reply:
Checked, the optimizer with uperhead is different from the fpn setting.